collections/the university of manchester/Persian_MS_300.xml

```diff
@@ -18,7 +18,8 @@
                         <settlement>Manchester</settlement>
                         <postCode>M3 3EH</postCode>
                         <addrLine>
-                            <ref target="http://www.library.manchester.ac.uk/specialcollections/">The John Rylands Library</ref>
+                            <ref target="http://www.library.manchester.ac.uk/specialcollections/"
```

Comment:
This is also a strange bit of formatting -- to put the closing `>` on a new line.